test/unit/mdc-chips/mdc-chip-set.test.js

```diff
@@ -139,3 +140,11 @@ test('#adapter.appendChild adds a child to the chip set element', () => {
   component.getDefaultFoundation().adapter_.appendChild(chipEl);
   assert.equal(root.childNodes[3], chipEl);
 });
+
+test('#adapter.removeChip removes the chip object from the chip set', () => {
+  const root = getFixture();
+  const component = new MDCChipSet(root, undefined, (el) => new FakeChip(el));
+  const chip = component.chips[0];
+  component.getDefaultFoundation().adapter_.removeChip(chip);
+  td.verify(chip.remove());
```

Comment:
Also veriy that component.chips' length is 0?

Reply:
Done. (It should have length 2 since we're only removing 1 chip)